package_manifest.yaml

```diff
@@ -22,14 +22,18 @@ foreman_scl_packages:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
     repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_target_repos:
+      - downloaded_rpms
```

Comment:
Previously this was 'hidden' as a nuance of how Obal handled scratch builds. This seems strange to have to surface here? Could Obal just include this as a default lookup location?

Reply:
Obal certainly could. We have configs (see below for repoclosure only examples) where we don't have a downloaded_rpms folder. What would happen if we still add it in that case. We could make obal only add it if present?

Reply:
Implemented in https://github.com/theforeman/obal/pull/186